sphinx/builders/html/__init__.py

```diff
@@ -1035,8 +1035,21 @@ def hasdoc(name: str) -> bool:
             templatename = newtmpl
 
         # sort JS/CSS before rendering HTML
-        ctx['script_files'].sort(key=lambda js: js.priority)
-        ctx['css_files'].sort(key=lambda js: js.priority)
+        try:
+            ctx['script_files'].sort(key=lambda js: js.priority)
```

Comment:
Unrelated to this PR but you may consider just using something like
```suggestion
            get_priority = operator.attrgetter('priority')
            ctx['script_files'].sort(key=get_priority)
```
in the future.

Reply:
+1: I'll do it later.